crates/host/src/wasmbus/mod.rs

```diff
@@ -1502,7 +1505,8 @@ impl Host {
         ) {
             (hash_map::Entry::Vacant(_), None) => {}
             (hash_map::Entry::Vacant(entry), Some(count)) => {
-                let actor = fetch_actor(&actor_ref)
+                // FIXME: avoid duplicate actor fetch
```

Comment:
Can we link to an issue?

Reply:
Will file

Reply:
https://github.com/wasmCloud/wasmCloud/issues/495